purchase_location_by_line/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :alt: License: AGPL-3
+
+=========================
+Purchase Location by Line
+=========================
+
+Allows to define a specific destination location on each Purchase Order line.
+When the PO is confirmed, it will generate one Incoming Shipment per
+combination of destination location and expected date indicated in the
+Purchase Order Lines.
+
+This module is used to send the products to different locations, that could
+not be children of the default location of the same PO picking type. Not be confused.
```

Comment:
Do you mean that users can only put children locations of the default location of the picking type?

Reply:
Reading comprehension :astonished:

Reply:
:stuck_out_tongue: